test-utils/test-utils-domain/src/test/java/uk/gov/justice/services/example/cakeshop/domain/Ingredient.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.services.example.cakeshop.domain;
```

Comment:
I think its confusing to have copies of the example context domain here. We should create specific test cases for the Cucumber tests instead.